pandas/tests/test_downstream.py

```diff
@@ -150,6 +150,12 @@ def test_missing_required_dependency():
     # https://github.com/MacPython/pandas-wheels/pull/50
 
     pyexe = sys.executable.replace("\\", "/")
+
+    call = [pyexe, "-c", "import pandas;print(pandas.__file__)"]
+    output = subprocess.check_output(call).decode()
```

Comment:
umm can you explain what this is doing and why this works

Reply:
I guess it's more why the test doesn't work when pandas is a site package. there are comments on L146-L150. but could add more.